ui/packages/catalog-ui-search/src/main/webapp/react-component/container/marionette-region-container/marionette-region-container.tsx

```diff
@@ -41,10 +42,13 @@ export default hot(module)(
         this.region.show(this.props.view, {
           replaceElement: this.props.replaceElement,
         })
+        this.props.bindView && this.props.bindView(this.props.view)
```

Comment:
This is here to allow for calling methods on a view, once it has been instantiated. In this context doing things like `view.turnOnEditing()` etc which can't be accomplished without first refactoring the downstream Marionette components first.

Reply:
I'm going to disagree that we need this at the moment.  Since the marionette-region-container allows passing in an already instantiated view, that means you can accomplish this without this extra code.  I'll add a suggestion below that shows how this could be accomplished.

I also showed @samuelechu an example of how to do this the other day, so he could probably tag up with you and help convert it necessary.